eav/models/entity.py

```diff
@@ -113,7 +115,12 @@ def save(self):
                     and attribute_value is not None
                 ):
                     attribute_value = EnumValue.objects.get(value=attribute_value)
-                attribute.save_value(self.instance, attribute_value)
+                if commit:
+                    attribute.save_value(self.instance, attribute_value)
+                values[attribute.slug] = attribute_value
+        if not commit:
+            attributes_value = self.save_bulk(values, attributes)
+            Value.objects.bulk_create(attributes_value)
```

Comment:
If `commit=False`, I would expect *nothing* to be saved to the database, which is line with Django behavior. Instead, what do you think about returning a `list[Value]` back? This would work for any number of attributes that are updated during an `eav.save()` call.